### ISSUE TITLE
Add a comment once e2e finishes

### ISSUE THREAD

**Issue by zroubalik** [kedacore/keda#5308]
### Proposal

In order to be able to tell which e2e test run has passed, we should generate a new comment (instead of just adding an emoji to the original comment with `/run-e2e` command. The new comment should quote the original comment with `/run-e2e` command.

The benefit is, that we got a notification that e2e test run has finished, so we can check the results.

### Old Proposal

In order to be able to tell which PR is ready for final review/merge, it would be nice if a label (eg `e2e-passed`) could be added to the PR.

If the there are additional commits/pushes to the PR, the label should be automatically removed.

**Comment by JorTurFer** [comment]
Isn't it enough just the CI check?

**Comment by zroubalik** [comment]
Maybe 🤔 

I thinking how to do a review properly, once I trigger e2e test on a couple of PRs, I want to get back to these once they finish.

**Comment by JorTurFer** [comment]
Currently we can see if e2e has passed (or being skipped) here:
![image](https://github.com/kedacore/keda/assets/36899226/204c4572-e319-4a4a-9b86-0e5d43df51dd)

**Comment by zroubalik** [comment]
Yeah I know, maybe this is more about me, knowing when a new e2e test execution finished, so I can review the result. Sometimes it is hard to keep track. I should probably solve this on my side, the label is an overkill.

**Comment by zroubalik** [comment]
@JorTurFer what would actually help is to get some kind of notification once the e2e run finishes. Now it just updates a comment which is not something that generates notifications.

**Comment by JorTurFer** [comment]
Actually, the notification is sent on every comment

**Comment by JorTurFer** [comment]
We could try to notify somehow to a slack channel. Could it help? If yes, we have to request the slack channel in kubernetes workspace

**Comment by zroubalik** [comment]
> Actually, the notification is sent on every comment

It is not sent on comments updates, or am I mistaken? I am interested in getting a notification once e2e test run finishes.

Slack notification is probably an overkill.

**Comment by JorTurFer** [comment]
sample

**Comment by tomkerkhove** [comment]
> > Actually, the notification is sent on every comment
> 
> It is not sent on comments updates, or am I mistaken? I am interested in getting a notification once e2e test run finishes.
> 
> Slack notification is probably an overkill.

I can see value in a comment post or so indeed

**Comment by zroubalik** [comment]
Modified the proposal to create a new comment, instead of just a label.

**Comment by tomkerkhove** [comment]
I think we should use the same for starting e2e tests instead of updating the message with test run link given that does not trigger GitHub notification, but it might be just me.

**Comment by samuraixwandering** [comment]
@zroubalik this looks like it is mostly done already, though nothing links back here so it is easy to miss.

#7760 and #7809 added the test summary to the `/run-e2e` comment, so points 3 and 4 from your comment above are covered: the comment now carries pass and fail counts, the actual test names, and setup or teardown failures.

What is still missing is the part the issue was opened for. The summary is written with `comment-id`, so it edits the original `/run-e2e` comment instead of posting a new one, and edits do not generate a notification. On #7941 for example the comment was created at 05:48 and last updated at 07:22 when the results landed, so the only notification anyone got was from the person typing the command in the first place.

The change looks like one line in `pr-e2e.yml`, swapping `comment-id: ${{ github.event.comment.id }}` for `issue-number: ${{ needs.triage.outputs.pr_num }}` on the `Append test results to comment` step, plus a line in the body linking back to the original comment to cover point 2.

@wozniakjan since that is your code from #7760, would you rather keep appending? Happy to send the change if a separate comment is still what you want, or to leave it alone if appending has worked out better in practice.

**Comment by JorTurFer** [comment]
I think this is done indeed,. isn't @wozniakjan ?
